recipes/solaris-gpu/meta.yaml

```diff
@@ -0,0 +1,117 @@
+{% set name = "solaris" %}
+{% set version = "0.1.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 5b357c60c87bbbb79dd8386f94be6432cf205ea7b44b4959a161d476eb67861b
+
+build:
+  number: 0
+  skip: true  # [py<36]
+  entry_points:
+    - geotransform_footprints = solaris.bin.geotransform_footprints:main
+    - make_graphs = solaris.bin.make_graphs:main
+    - make_masks = solaris.bin.make_masks:main
+    - mask_to_polygons = solaris.bin.mask_to_polygons:main
+    - spacenet_eval = solaris.bin.spacenet_eval:main
+    - solaris_run_ml = solaris.bin.solaris_run_ml:main
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vv "
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - {{ compiler('c') }}
+    - affine >=2.2.2
+    - albumentations >=0.2.3
+    - fiona >=1.8.6
+    - gdal >=2.4.0
+    - geopandas >=0.4.0
+    - matplotlib >=3.1.0
+    - networkx >=2.2
+    - numpy >=1.15.4
+    - opencv >=4.1.0
+    - pandas >=0.23.4
+    - pip
+    - python
+    - pyyaml >=5.1
+    - rasterio >=1.0.18
+    - rio-cogeo >=1.0.0
+    - rtree >=0.8.3
+    - scikit-image >=0.14.0
+    - scipy >=1.2.0
+    - shapely >=1.6.4
+    - tensorflow >1.10,<=1.13.1
+    - pytorch >=1.0.1
+    - torchvision >=0.3.0
+    - tqdm >=4.32.2
+    - urllib3 >=1.23, <1.25.3
+  run:
+    - affine >=2.2.2
+    - albumentations >=0.2.3
+    - fiona >=1.8.6
+    - gdal >=2.4.0
+    - geopandas >=0.4.0
+    - matplotlib >=3.1.0
+    - networkx >=2.2
+    - numpy >=1.15.4
+    - opencv >=4.1.0
+    - pandas >=0.23.4
+    - python
+    - pyyaml >=5.1
+    - rasterio >=1.0.18
+    - rio-cogeo >=1.0.0
+    - rtree >=0.8.3
+    - scikit-image >=0.14.0
+    - scipy >=1.2.0
+    - shapely >=1.6.4
+    - tensorflow-gpu >1.10, <=1.13.1
+    - cuda90
+    - pytorch >=1.0.1
+    - torchvision >=0.3.0
+    - tqdm >=4.32.2
+    - urllib3 ==1.24.3
```

Comment:
These pins are excessive. When shipping a library, only the minimally absolutely necessary pins should be used. Otherwise, your package won't be able to coexist in envs with others.

Reply:
I can loosen where it won't interfere with functionality, thanks